### DISCUSSION TITLE
Tabela de Relacionamentos

### DISCUSSION THREAD

**Discussion by BrunoCarolino7** [balta-io/forum#330] [Tira Dúvidas] [answered]
Fala, Professor! Bom Dia ! Espero que esteja bem !

Curso - > Fundamentos do EntityFramework

Estou com uma dúvida, se o senhor puder ajudar , pf :)

durante o curso, ao construir o (De - Para) vi que vc não colocou as tabelas de relacionamentos (nxn) no DbSet<> nem as criou na pasta "Models". Não é necessário criar classes no C# referenciando essas tabelas ou não se deve mesmo fazer isso ?

Valeu, André!

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @BrunoCarolino7, como vai?
Não há necessidade de criar, o EF já entende e cria a tabela. Pode conferir no banco :)

**Reply by BrunoCarolino7** [reply]
Tudo joia por aqui. 

Valeu pela resposta, André!
